Source/Charts/Highlight/ChartHighlighter.swift

```diff
@@ -48,19 +48,15 @@ open class ChartHighlighter : NSObject, IHighlighter
     /// - returns:
     @objc open func getHighlight(xValue xVal: Double, x: CGFloat, y: CGFloat) -> Highlight?
     {
-        guard let chart = chart
-            else { return nil }
+        guard let chart = chart else { return nil }
         
         let closestValues = getHighlights(xValue: xVal, x: x, y: y)
-        if closestValues.isEmpty
-        {
-            return nil
-        }
+        guard !closestValues.isEmpty else { return nil }
```

Comment:
This guard is harder to read than how it was before. I have to think about it for a second when I try to understand this line. It was more straightforward before. It doesn't reduce nesting so we can switch it back.

Reply:
While `guard` often does reduce nesting, it's purpose is to indicate that
> You use a guard statement to require that a condition must be true in order for the code after the guard statement to be executed. [Apple](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/ControlFlow.html).

It seems reasonable to me to expect reading "only continue if `closestValues` is not empty"